src/ui/src/session.cpp

```diff
@@ -147,6 +147,7 @@ std::vector<WindowSession> Session::windowSessions()
     const auto& sessionWindows = session.windows();
 
     std::vector<WindowSession> windows;
+    windows.reserve( static_cast<size_t>( sessionWindows.size() ) );
```

Comment:
Another settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3